regolith/helpers/l_milestoneshelper.py

```diff
@@ -215,8 +224,14 @@ def sout(self):
                     print(f"    Notes:")
                     for note in ms.get("notes"):
                         print(f"      - {note}")
+                if ms.get("uuid"):
```

Comment:
remove from the if statement because uuid is required in the schema.   We want it to crash if a uuid is missing because the data are corrupted.